src/index.ts

```diff
@@ -44,6 +44,10 @@ export class MtLinkSdk {
     };
   }
 
+  public setSamlSubjectId(value: string): void {
```

Comment:
Line 40:

`this.storedOptions` always gets overridden by `const { mode = 'production', ...rest } = options;` so just wondering why we extend from it.